### ISSUE TITLE
sbt-converter not found on Maven

### ISSUE THREAD

**Issue by peterbanda** [ScalablyTyped/Converter#406]
Hi there,
The `"org.scalablytyped.converter" % "sbt-converter"`plugin (artifact) doesn't seem to be available on [Maven](https://mvnrepository.com/artifact/org.scalablytyped.converter) and additionally the `Bintray` repo has been permanently shut down (although it's still referenced in the tutorial / docs). Could you pls. release `sbt-converter` to Maven or any other public location?

Thanks a ton,
:Peter

**Comment by kazievab** [comment]
You can find it [here](https://search.maven.org/artifact/org.scalablytyped.converter/sbt-converter/1.0.0-beta37/jar)

**Comment by oyvindberg** [comment]
Where did you copy instructions from @peterbanda ? Another place than here https://scalablytyped.org/docs/plugin ?

Everything was in bintray before, but after the bintraypocalypse everything was moved to maven central

**Comment by peterbanda** [comment]
Yes, I found it, thanx for the clarifications!

It's indeed in maven from the version [beta-31 to beta-37](https://repo1.maven.org/maven2/org/scalablytyped/converter/sbt-converter_2.12_1.0/). What confused me initially was the `_1.0` suffix.

@oyvindberg: I was actually trying to run your example project [mbox-slinky](https://github.com/ScalablyTyped/mobx-slinky), when I've realized the `sbt-converter` cannot be pulled. I've managed to get it running by changing the `sbt.version` to `1.5.8` and `sbt-converter` to `1.0.0-beta37` (without `Resolver.bintrayRepo("oyvindberg", "converter")`). I can submit a pull request if needed.

Cheers
